src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -2061,7 +2063,7 @@ private void invalidateNode(UUID studyUuid, UUID nodeUuid, UUID rootNetworkUuid)
         invalidateNodeInfos.setNetworkUuid(rootNetworkService.getNetworkUuid(rootNetworkUuid));
 
         deleteInvalidationInfos(invalidateNodeInfos);
-
+        rootNetworkNodeInfoService.clearNodeExportNetworks(nodeUuid, rootNetworkUuid);
```

Comment:
Move this into invalidateRootNetworkNode(RootNetworkNodeInfoEntity rootNetworkNodeInfoEntity, InvalidateNodeTreeParameters invalidateTreeParameters) of RootNetworkNodeInfoService.java